client/src/modules/plan/plan.css

```diff
@@ -0,0 +1,6 @@
+.plan-container {
+  display: flex;
+  flex-direction: column;
+  height: 75vh;

```

Comment:
What's this for?

check with the new global styles, should be able to have your top-most container have height: 100% now